### PR TITLE
feat: get response.body as a stream with the fetch API

### PR DESCRIPTION
**Pull request by Fine0830**
Intercept fetch requests and fetch a stream from the fetch API. Reference https://github.com/apache/skywalking/issues/6546

Screenshots
fetch
res
segments
Here is a demo for getting response.body as a stream with fetch.  https://github.com/SkyAPMTest/skywalking-client-test

Signed-off-by: Qiuxia Fan <qiuxiafan@apache.org>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,7 +60,6 @@
     "web-performance"
   ],
   "dependencies": {
-    "js-base64": "^3.6.0",
-    "whatwg-fetch": "^3.5.0"
```
Comment (wu-sheng) [inline package.json:64]: Are you implementing `fetch` by ourselves?

Comment (wu-sheng) [reply package.json:64]: Are we totally rewriting the thing? And the IP is clear?

Comment (Fine0830) [reply package.json:64]: I just extended functions to the `window.fetch` to intercept http requests with fetch. I think there are no problems of IP.

Comment (Fine0830) [reply package.json:64]: `window.fetch` is the browser's own function.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**